.travis.yml

```diff
@@ -110,5 +125,5 @@ matrix:
 
 script: ./runTests.py -j2 $TESTFOLDER
 
-sudo: false
+sudo: true
```

Comment:
@seantalts, any opinions on changing `sudo` to `true`? Although... it looks like this should be `required`?
https://docs.travis-ci.com/user/reference/overview/

Reply:
It'd be nice if we could avoid sudo - it makes each VM spin up take ~5 minutes more to run, and we run into a lot of time limits already. Like if we could on Travis use a compiler flag to add to the library path instead of a symlink requiring sudo that could be good.

Reply:
^I'll look into how to do that